src/internal/components/dropdown/index.tsx

```diff
@@ -283,12 +283,12 @@ const Dropdown = ({
      * finish bubbling, so that it is not immediately captured here.
      */
     const timeout = setTimeout(() => {
-      window.addEventListener('click', clickListener);
+      window.addEventListener('mousedown', clickListener);
```

Comment:
This means that when clicking on the input when the dropdown is open, the dropdown temporarily disappears and then reappears. I'm not sure if this is problematic as such, but it's definitely a change from the current behavior and seems potentially detrimental. Have you discussed with design?

Reply:
Nice catch! I will check if it is possible to avoid having that effect.